openquake/mbt/tests/tools/fault_modeler/test_fault_modeling_utils.py

```diff
@@ -247,24 +247,39 @@ def test_calc_mfd_from_fault_params_double_gr(self):
 
         # mfd_type should be set to 'DoubleTruncatedGR' by default;
         # this is part of the test.
+        # 20190801 - added M_ref
+        # mdf_rates modified, using M_ref = 0 as default value
 
         mfd, seis_rate = fmu.calc_mfd_from_fault_params(
                                                 self.fault_1,
-                                                # mfd_type='DoubleTruncatedGR',
+                                                #mfd_type='DoubleTruncatedGR',
```

Comment:
This is probably a leftover and should be removed

Reply:
ok